docs/Symbols.md

```diff
@@ -0,0 +1,98 @@
+# Symbols
+
+## Symbols filter settings

```

Comment:
I have a slight preference for 'Symbol filters'

Reply:
To be consistent with Formatting, I replaced with  `xml.symbols.filters`